paper.md

```diff
@@ -61,49 +60,48 @@ Fig 1. The four lessons in the curriculum for a metagenomics workshop.
+To promote an interactive learning experience, we use a collaborative live document and small groups to do exercises and discussions. In this way, there are fewer barriers for student’s participation, and they gain practical experience of the different solutions bioinformatic problems can have. We include bonus exercises around the lessons that are not meant to be solved by all the learners but are an opportunity for advanced participants to tackle more challenging tasks. They help to leverage the background and learning speed of the learners and provide opportunities to practice and reinforce the content learned. Another strength of our workshop structure is introducing programming languages followed by a practical example of their usage, so the metagenomic analysis grows while the coding skills learnt become meaningful. The lessons are helpful even if the student’s primary goal is to learn coding rather than metagenomics. We ask the attendees to fill surveys before, during, and after the workshops to adapt our teaching strategies to the current and future learning groups. With the results of these surveys, we have been able to improve the curriculum’s content, explanations, and exercises.
 
 # Story of the Project
-Creating a lesson that comprised all of the steps required to process and analyze metagenomics data came to be when one of us wanted to learn to understand metagenomics data but found it overwhelming even to decide where to start. Being someone who already used and knew the advantages of the resources offered by The Carpentries, she acknowledged the absence of a curriculum about metagenomics on this platform. When she learned of the project of Carpentries Incubator, she reached out to a fellow student that had experience on the subject and started building the episodes. Little by little other students in the same institute, who were also working on metagenomics, started contributing with the pipelines they were learning. Other team members were recruited as helpers after attending a workshop and later began collaborating to develop the episodes.
+The idea to create a lesson comprising all of the steps required to process and analyze metagenomics data arose when one of the development team wanted to learn to understand metagenomics data but found it overwhelming even to decide where to start. As someone who already used and knew the advantages of the resources offered by The Carpentries, she recognised the absence of a curriculum about metagenomics on this platform. When she learned of the project of The Carpentries Incubator, she reached out to a fellow student that had experience on the subject and started building the episodes. Other students in the same institute, who were also working on metagenomics, started contributing with the pipelines they were learning. Further team members were recruited as helpers after attending a workshop and later began collaborating to develop the episodes.
```

Comment:
If there is space, I think it would be good to add a sentence here about why you chose to base two of the lessons on the existing Data Carpentry Genomics curriculum.